### PR TITLE
:pencil: fix clippy warnings and test_kv_destroy (#2)

### PR DESCRIPTION
**Pull request by Xudong-Huang**

### CODE REVIEW

File: src/kvtable.rs [hunk 1]
```diff
@@ -70,7 +70,7 @@ impl<K, V> Drop for KVs<K, V> {
 }
 
 // ---Structure for resizing -------------------------------------------------------
-
+#[derive(Default)]
```
Comment (rlei) [inline src/kvtable.rs]: I think it's fine not to implement `Default` for `CHM` as it should not be made public anyway. I'll change its visibility later.

File: src/lib.rs [hunk 2]
```diff
@@ -597,6 +604,7 @@ impl<K: Eq + Hash, V: Eq> NonBlockingHashMap<K, V> {
         }
     }
 
+    #[allow(dead_code)]
```
Comment (rlei) [inline src/lib.rs]: Let's just keep the clippy warnings for these two functions (`fast_keyeq` and `print_table`) as I planned to revisit them later.

**Review by rlei — COMMENTED** [review]
Thanks a lot for the PR, I have added two comments :)

BTW could you remove `:pencil:` from the PR title? Thanks!

**Comment by rlei** [comment]
hi @Xudong-Huang any update?

**Comment by Xudong-Huang** [comment]
do you want me just remove those lines in the comment?

**Pull request merged**